_posts/2019-02-26-bazel-0.23.md

```diff
@@ -0,0 +1,211 @@
+---
+layout: posts
+title: "Bazel 0.23"
+authors:
+  - laurentlb
+---
+
+[Bazel 0.23.0](https://github.com/bazelbuild/bazel/releases/tag/0.23.0) has
+arrived with an exciting mix of new features, important bug fixes, and updated
+documentation.
+
+This version of Bazel has been regression-tested against 51 open-source
```

Comment:
is there a link to a full list of projects?

Reply:
Is it useful to add this link?
https://buildkite.com/bazel/bazel-at-head-plus-downstream/builds/821

Reply:
yes! do we have a fully green run?

Reply:
The tests for 0.23.0rc2 is fully green, 0.23.0rc3 has one failure.
Which one do you prefer?

Reply:
hmm, I think it doesn't matter that much. Let's go with the rc3.